docs/content/installation/integrations/app-protect-waf/compile-waf-policies.md

```diff
@@ -0,0 +1,358 @@
+---
+docs: DOCS-000
+title: Compile NAP WAF policies using NGINX Instance Manager
+toc: true
+weight: 300
+---
+
+## Overview
+
+This guide describes how to use F5 NGINX Instance Manager to compile NGINX App Protect WAF Policies for use with NGINX Ingress Controller.
```

Comment:
General question: do we follow the principles of "Every page is page one"? If so, we should spell out `web application firewall` on the first mention on every page.

Reply:
NGINX App Protect WAF is the product name.